### PR TITLE
Add MIT license text

### PR DESCRIPTION
**Pull request by kasicka**
Hi, 
we would like to package your module for fedora and we need to have license included in the tarball. Could you, please, merge this PR and preferably update the npmjs tarball as well?
Thank you!

### CODE REVIEW

**Comment by phadej** [comment]
:+1:

**Comment by kemitchell** [comment]
@kasicka I'll be happy to include MIT if that's what Fedora needs, even though I think the package metadata is sufficient. Thanks for caring about licensing! License audit is yeoman's work :smile:

As a lawyer, I'm curious about the Fedora policy. Might you link to the Fedroa repo guidline doc? Thanks!

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2015 Kyle E. Mitchell
```
Comment (kemitchell) [inline LICENSE:3]: += " and other contributors"

Would be nice if you could amend. Thanks!

**Comment by piotr1212** [comment]
Hi @kemitchell The license text requirement is actually enforced from the MIT license itself, it literally says:

```
The above copyright notice and this permission notice shall be included 
in all copies or substantial portions of the Software.
```

I think this is some leftover from the old days before the internet when you would send you code on floppy's and there would be no way for the receiving party to get the actual license text.

**Comment by kemitchell** [comment]
Hah! You just one-upped a full-time licensing lawyer.

If @kasicka would be so kind as to amend per my line-comment, I'll merge.

**Comment by kemitchell** [comment]
I went ahead and added myself. Thanks again to all.

**Comment by kemitchell** [comment]
Published to registry.npmjs.org as spdx@0.4.2, commit d54d741609c6b99dff484d335926548996d3520a

**Comment by kemitchell** [comment]
~~@othiym23 I've just published a semver-patch update to spdx.js with a LICENSE file, which the Fedora people need for packaging. Anything special I need to do to get spdx@0.4.2 bundled with the next releases of v2 and v3?~~

So sorry. npm is already using -expression-parse. I'll be quiet now.

**Comment by kemitchell** [comment]
On a related note, I hope to eventually move npm over to [spdx-expression-parse](https://www.npmjs.com/packages/spdx-expression-parse), which is both smaller and more refined.

**Comment by kemitchell** [comment]
@kasicka and @piotr1212: Newer versions of npm package https://github.com/kemitchell/spdx-expression-parse.js instead. I have added a copy of MIT there, as well.

**Comment by othiym23** [comment]
This week's releases are as light as I can make them due to the holiday, but if the license is the only thing that's changed, I can probably get it in to both `npm@2` and `npm@3`.

**Comment by kemitchell** [comment]
Thanks @othiym23. No rush from my point of view, but I can't speak for the Fedora people. I'd imagine their processes take time, too.

You've a lot on your hands. I really, _really_ appreciate being able to at-mention you on these things.

**Comment by othiym23** [comment]
@kemitchell Did you publish `spdx@0.4.2`? `npm` includes that directly as well, and I'm not seeing the new version on the registry. (Unless you publish it in like the next 10 minutes, it won't make it for this week's release, but we can get it in next week.)

**Comment by kemitchell** [comment]
Hmm. Next week, then. Sorry. Not sure what happened there.

**Comment by kemitchell** [comment]
0.4.2 published. https://www.npmjs.com/package/spdx

My bad, Fedora-folk. There's a gremlin somewhere in my shellscript!

**Comment by piotr1212** [comment]
No worries ;) thanks!

**Comment by othiym23** [comment]
This prompted me to take a closer look at `npm@2`, and it turns out we've been including `spdx` completely superfluously. I removed it and made sure the SPDX handling code we _do_ use is up to date. @piotr1212, this may be relevant to your interests.

**Comment by kemitchell** [comment]
@othiym23, is v2 doing normalize-package-data to validate-npm-package license to spdx-expression-parse?

I need to read the release process doc again.

**Comment by kemitchell** [comment]
@kasicka I accidentally included some functional changes in addition the MIT License in 0.4.2. You will want 0.4.3, which I've just published.

**Pull request closed without merging**